Add tutorials guidelines in the main tutorials branch (#1950)
ansys/pydpf-core: doc/source/getting_started/write_doc/tutorial_structure_template.rst

.. _ref_tutorial_template:

==============
Tutorial title
==============

> .. include:: ../../../links_and_refs.rst

Author: ```suggestion

```

> .. |Examples| replace:: :class:`ansys.dpf.core.examples`

Reviewer: ```suggestion
```
Already in `links_and_refs.rst`.

> This sentence resumes the goal of the tutorial

Reviewer: ```suggestion
This sentence describes the goal of the tutorial.
```

Reviewer: @luisaFelixSalles my bad, is that actually true?

Author: Yes. Like, this is the same sentence you put in the card

Author: ```suggestion
A single sentence describing the goal of the tutorial. Same sentence of the tutorial card in the section page.
```

> Introduction to the tutorial

Reviewer: ```suggestion
A full description of the tutorial.
```

Author: ```suggestion
Introduction to the tutorial. Here, you provide the necessary context or foundational information for understanding the tutorial.
```

> :jupyter-download-script:`Download tutorial as Python script<file_name>` :jupyter-download-notebook:`Download tutorial as notebook<file_name>`

Author: ```suggestion
:jupyter-download-script:`Download tutorial as Python script<file_name>` :jupyter-download-notebook:`Download tutorial as Jupyter notebook<file_name>`
```

First Step
----------

First, you ...

.. jupyter-execute::

    # Code block 1

Second step
-----------

Then, you ...

.. jupyter-execute::

    # Code block 2

Final Step
----------

Finally,  you ...

.. jupyter-execute::

>     # Code block 3

Reviewer: We want a blank line at the end of files.
